### ISSUE TITLE
compatibility with soul fire´d

### ISSUE THREAD

**Issue by slovinsqui** [Ladysnake/Impaled#44]
I just noticed the soulfork sets enemies on normal fire instead of blue fire
there´s a mod called soul fire´d that has an API for compatibility for that, it would be cool if you could add compatibility with it

https://www.curseforge.com/minecraft/mc-mods/soul-fired

**Comment by Eman-3600** [comment]
If they were to add that, they'd more likely do it for [https://www.curseforge.com/minecraft/mc-mods/on-soul-fire](https://www.curseforge.com/minecraft/mc-mods/on-soul-fire) instead.

**Comment by Crystal-Spider** [comment]
However my mod makes it way easier to add compatibility for ;) Also my mod adds enchantments and the possibility to have a whole ecosystem of fires registered by other mods, possibly allowing in the future to have different tridents that set on different kind of fire. Furthermore it has a more consistent fire behavior.  
Of course I know that 5M downloads vs 30k is a big difference. However consider the ease of use, that maybe by adding Soul Fire'd as an optional dependency it will help get more widespread and the potential long term investment, I'd at least consider it!

P.S. I take this comment to mention [this issue](https://github.com/Nyphet/soul-fire-d/issues/19) to track its progresses.

**Comment by Eman-3600** [comment]
Yeah perhaps. Honestly, a better thing to ask for is the ability to import it into a project to make addons.

**Comment by ImWhiff** [comment]
Me three begs the Soul fork to be compatible with Soul Fire'd 👉👈
